db/controllers/app.controllers.js

```diff
@@ -68,4 +68,12 @@ exports.deleteComment = (request, response, next) => {
         response.status(204).send({deletedComment})
     })
     .catch(next)
+}
+
+exports.getUsers = (request, response, next) => {
+    fetchUsers()
+    .then((users) => {
+        response.status(200).send({users})
+    })
+    .catch(next)
```

Comment:
There are no errors to catch on this endpoint so please remove this catch block.